EIPS/eip-2771.md

```diff
@@ -0,0 +1,185 @@
+---
+eip: 2771
+title: Secure Protocol for Native Meta Transactions
+author: Ronan Sandford (@wighawag), Liraz Siri (@lirazsiri),  Dror Tirosh (@drortirosh), Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Hadrien Croubois (@Amxx), Tomar Sachin (@tomarsachin2271), Patrick McCorry (@stonecoldpat), Nicolas Venturo (@nventuro), Fabian Vogelsteller (@frozeman) 
+discussions-to: https://github.com/ethereum/EIPs/pull/2771
+status: Draft
+type: Standards Track
+category: ERC
+created: 2020-07-01
+---
+
+## Simple Summary
+
+We define a minimal contract-level protocol for receiving a meta transaction
+through a trusted forwarder contract that helps a recipient contract avoid
+confusing the address of the transaction relayer with the address of the
+transaction signer. 
```

Comment:
You probably should shorten this up a bit and leave some content for the abstract.  😄 
```suggestion
A contract interface for receiving meta transactions through a trusted forwarder.
```